icefall/graph_compiler.py

```diff
@@ -29,6 +29,7 @@ def __init__(
         lexicon: Lexicon,
         device: torch.device,
         oov: str = "<UNK>",
+        need_repeat_flag: bool = False,
```

Comment:
You may also need to modify the corresponding `train.py` to pass True to this funciton.

Reply:
I think we only have bpe models that support delay-penalty, @yaozengwei  Did you update the phone-based recipes?

Reply:
I am meaning that will your current change break any existing recipes?

Reply:
I think it won't break any recipes,  will check it again.

Reply:
Previously, the repeat attr is always added. But this PR removes it since the arg is always False.

Reply:
Yes, you are right, we never tried the phone-based models, but it did support in our code. Will fix it.

Reply:
> that

Yeah. I also updated the phone-based graph-compiler.